src/style.css

```
@@ -1 +1,52 @@
-/* add styles here */
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  background-image: url('./images/star-active.svg');
+  background-repeat: no-repeat;
+  background-position: center center;
+  background-size: 13px;
+}
+
+.stars--0 .stars__star:nth-child(n) {
```

```suggestion
.stars--0 .stars__star:nth-child(n),
...,
.stars--2 .stars__star:nth-child(n+3) {
```
Consider using commas to group your selectors